### ISSUE TITLE
DateTimePicker testing with jest : Cannot read property 'getDefaultDisplayValue' of undefined

### ISSUE THREAD

**Issue by geoffreyLB** [react-native-datetimepicker/datetimepicker#510]
# Bug report

## Summary

I got this error when I running tests with `Jest`
![Capture d’écran 2021-09-20 à 18 24 56](https://user-images.githubusercontent.com/24936997/134038544-6fd64f92-ab2f-4d39-b326-bb6c8a283bad.png)

# Steps to reproduce

1. npm install @react-native-community/datetimepicker
2. npm install @testing-library/react-native && jest
3. add screen with `<DateTimePicker />` and test your component

**stdout**:  Error: Uncaught [TypeError: Cannot read property 'getDefaultDisplayValue' of undefined]

I take a look on the source code, and it seems that in `layoutUtilsIOS.js` file,  `NativeModules.RNDateTimePickerManager` is `undefined` in test environment, I don't know why. 

<img width="961" alt="Capture d’écran 2021-09-20 à 18 42 29" src="https://user-images.githubusercontent.com/24936997/134040914-e3a04fd4-7034-47cc-ac40-6772a05b0e5d.png">

The only solution I found is to mock datetimepicker [solution](https://stackoverflow.com/questions/60271608/issue-testing-the-datetimepickermodal-with-jest-react-native-testing-library)
```js 
jest.mock('@react-native-community/datetimepicker', () => {
  const React = require('react');
  return class MockPicker extends React.Component {
    render() {
      return React.createElement('DatePicker', { date: '21.07.2020' });
    }
  };
});
```

## Environment info

datetimepicker version: 3.5.2
jest: "^27.2.1",

**Comment by vonovak** [comment]
:tada: This issue has been resolved in version 6.3.0 :tada:

If this package helps you, consider [sponsoring us](https://github.com/sponsors/react-native-datetimepicker)! :rocket:

**Comment by vishaldhanotiyapersistent** [comment]
https://github.com/react-native-datetimepicker/datetimepicker/issues/216#issuecomment-741794280
This helps me to resolve my issue.
